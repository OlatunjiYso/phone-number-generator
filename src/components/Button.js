import React from 'react';


function Button(props) {
  const { label, action} = props;
  return (
    <button onClick={action}>
      {label}
    </button>
  );
}

export default Button;
