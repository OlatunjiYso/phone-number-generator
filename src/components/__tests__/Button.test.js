import React from 'react';
import { shallow } from 'enzyme';
import Button from '../Button';

it('renders the Button component', () => {
  const wrapper = shallow(<Button />);
  expect(wrapper).toMatchSnapshot()
});