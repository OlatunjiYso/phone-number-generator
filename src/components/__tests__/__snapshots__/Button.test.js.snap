// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the  component 1`] = `ShallowWrapper {}`;

exports[`renders the App component 1`] = `ShallowWrapper {}`;

exports[`renders the Button component 1`] = `ShallowWrapper {}`;
